module/checker.js

```diff
@@ -0,0 +1,42 @@
+import updateLocalStorage from './localstorage';
+
+const checker = (te, bo, to) => {
+// check
+  bo.forEach((e, n) => {
+    e.addEventListener('click', () => {
+      const parent = e.parentElement;
+      parent.classList.add('check');
+      const grand = e.parentElement.parentElement;
+      const line = grand.querySelector('#tasks');
+      line.classList.add('line');
+      grand.setAttribute('id', 'completed');
+
```

Comment:
- [ ] Good job, for easy understanding always make sure to give your values names that can be understood incase anyone picks up your code in the future